tests/tests_pytorch/strategies/test_ddp_integration.py

```diff
@@ -202,21 +202,18 @@ def training_step(self, batch, batch_idx):
         return super().training_step(batch, batch_idx)
 
 
-@pytest.mark.skipif(
-    # TODO: investigate threading issue in this configuration
-    _IS_WINDOWS,
-    # and (sys.version_info.major, sys.version_info.minor) == (3, 11)
-    # and compare_version("torch", operator.eq, "2.1.0", use_base_version=True)
-    reason="threading issue",
-)
-def test_find_unused_parameters_exception():
+def test_find_unused_parameters_exception_ddp_spawn():
     """Test that the DDP strategy can change PyTorch's error message so that it's more useful for Lightning users."""
     trainer = Trainer(accelerator="cpu", devices=1, strategy="ddp_spawn", max_steps=2)
     with pytest.raises(
         ProcessRaisedException, match="It looks like your LightningModule has parameters that were not used in"
     ):
         trainer.fit(UnusedParametersModel())
 
+
+@RunIf(min_cuda_gpus=1, standalone=True)
```

Comment:
This test doesn't use CUDA?

```suggestion
@RunIf(standalone=True)
```

Reply:
standalone doesn't run on cpu-only configuration last time I checked?